packages/angular_devkit/architect_cli/BUILD.bazel

```diff
@@ -13,6 +13,8 @@ ts_library(
     srcs = [
         "bin/architect.ts",
     ] + glob(["src/**/*.ts"]),
+    devmode_module = "commonjs",
```

Comment:
If `devmode_module` and `devmode_target` are set on all `ts_library` then it would be clearer to have a `/tools/defaults.bzl` which re-exports ts_library as a macro with these set. See angular/angular.

Reply:
Added macro